compose/compose-file/compose-file-v2.md

```diff
@@ -1510,8 +1510,8 @@ volume_driver: mydriver
 > entry in the [top-level `volumes` option](#volume-configuration-reference).
 
 
-See [Docker Volumes](/engine/userguide/dockervolumes.md) and
-[Volume Plugins](/engine/extend/plugins_volume.md) for more information.
+See [Docker Volumes](/storage/volumes.md) and
+[Volume Plugins](/engine/extend/plugins_volume/) for more information.
```

Comment:
@thaJeztah this entry has .md removed from the link. Is this intentional?

Reply:
Also, couldn't find the `/extend/` directory within `/engine/`. Do we autogenerate these files?

Reply:
Ah, yes, this is one of the files we fetch during the build.

I could change it back if you want, but it's also ok to use the "final" URL (without .md), as there's nothing to link to while working in this repository

Reply:
Thanks. Yes, not necessary to add the .md extension if the file aren't present.